app/models/card.rb

```diff
@@ -1,5 +1,7 @@
 class Card < ActiveRecord::Base
   belongs_to :card_set
+  validates :card_number, :presence, uniqueness: true
```

Comment:
Because of the `image_url` changes that Budrow and I worked on yesterday the `:card_number` validation needs to be removed. Name also must be present but won't be unique, even within a set! (There are multiple basic lands in each set, for example.) I'm not actually sure what we want to validate about cards. Let's brainstorm for just a minute.